include/field3d.hxx

```diff
@@ -233,6 +233,11 @@ class Field3D : public Field, public FieldData {
    * Ensure that yup and ydown refer to this field
    */
   void mergeYupYdown();
+
+  /*!
+   * Clear all yup/ydown fields and field_fa
```

Comment:
What do you mean by clear (delete, zero out, something else)?

Reply:
I mean delete (if they exist). It would make more sense to change the name...